package.json

```diff
@@ -28,21 +31,27 @@
   "author": "Kent C. Dodds <me@kentcdodds.com> (https://kentcdodds.com)",
   "license": "MIT",
   "dependencies": {
-    "@babel/runtime": "^7.9.2",
-    "babel-plugin-macros": "^2.8.0",
+    "@babel/runtime": "^7.12.5",
+    "babel-plugin-macros": "^3.0.1",
     "require-from-string": "^2.0.2"
   },
   "devDependencies": {
+    "@types/babel__core": "^7.1.12",
+    "@types/babel-plugin-macros": "^2.8.4",
+    "@types/require-from-string": "^1.2.0",
```

Comment:
Should these be actual dependencies for TS consumers?

Reply:
Only if we export any types that rely on them... Which I think we do.

Reply:
We'll review the built type definitions and move any that need to.

Reply:
Just a heads up for you that `babel-plugin-codegen` [has the same issue](https://github.com/kentcdodds/babel-plugin-codegen/blob/main/package.json#L50) (assuming we find some that should be actual dependencies)

Reply:
I'm pretty sure we actually don't need to worry because the only thing people actually import is the macro which doesn't expose any types but our own.

Reply:
IMHO only `@types/babel__core` needs to be moved:
```sh
$ grep import dist/*.d.ts
dist/helpers.d.ts:import type babelCore from '@babel/core';
dist/index.d.ts:import type babelCore from '@babel/core';
dist/object-to-ast.d.ts:import type babelCore from '@babel/core';
```
I will make a PR in the fork and update this PR after our internal review.

Reply:
That has been merged now